harness/determined/deploy/aws/deployment_types/vpc.py

```diff
@@ -42,10 +42,12 @@ class EFS(VPCBase):
 
 class Lore(VPCBase):
```

Comment:
should this classname be changed too?

Reply:
This is an internal reference, we are doing the naming migration in phases and focusing on user facing noes first.